packages/services/api/src/modules/target/module.graphql.ts

```diff
@@ -20,6 +20,10 @@ export default gql`
     updateTargetGraphQLEndpointUrl(
       input: UpdateTargetGraphQLEndpointUrlInput!
     ): UpdateTargetGraphQLEndpointUrlResult!
+    updateTargetDangerousChangeClassification(
+      target: TargetReferenceInput!
```

Comment:
I recall talk of moving to using the UUIDs, but existing mutations use the selectors.
I assumed the longer goal was to allow both everywhere using this `TargetReferenceInput`?
If not, I can adjust this.

Reply:
We always use a input type named after the field for mutation fields.

```suggestion
      input: UpdateTargetDangerousChangeClassificationInput!
```

```
input UpdateTargetDangerousChangeClassificationInput {
  target: TargetReferenceInput!
}
```